pyne/mesh.py

```diff
@@ -704,6 +704,16 @@ def __init__(self, mesh=None, structured=False,
             stored in self.dims.
 
         """
+        # if Mesh is made and no parameters passed, raise MeshError
+        if mesh is None and not structured and structured_coords is None and \
+            structured_set is None and structured_ordering=='xyz' and mats==():
+            raise MeshError("Trivial mesh instantiation "
```

Comment:
Incomplete thought...
Perhaps:
```suggestion
            raise MeshError("Trivial mesh instantiation detected. "
```